pkg/cri/containerd/runtime.go

```diff
@@ -0,0 +1,72 @@
+package containerd
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/containerd/containerd/oci"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+const k8sNamespace = "k8s.io"
+
+// ContainerdRuntime represents a connection to the containerd runtime
+type ContainerdRuntime struct {
+	client            Client
+	namespacedContext context.Context
+}
```

Comment:
```golang
type NamespacedContainerRuntime struct {
   containerRuntime ContainerRuntime
   namespacedContext context.Context
}
```

Reply:
At the end what I don't like is the `client` since it says nothing, so I was expecting to have `ContainerRuntime` as the interface for the and `NamespacedContainerRuntime` as this struct since it represents an association of a namespace and the controller-runtime.

Reply:
Renamed the client already to `containerRuntime`.